about.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <script src="https://kit.fontawesome.com/b39c945343.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <section class="about_main_section">
+        <!-- Mobile header -->
+        <header id="mobile_header">
+            <nav class="toggleNav">
+                <img src="assets/icons/union.png" alt="union icon" />
```

Comment:
Great design @Ndaboom 🤝

- [ ] Kindly add `alt` attributes to all images across all your HTML files. this is a good practice that can help with screen readers.